docs/getting-started-guides/ubuntu/operational-considerations.md

```diff
@@ -39,51 +43,11 @@ ssh -i ./.local/share/juju/ssh/juju_id_rsa ubuntu@<public-ip-of-juju-controller>
 
 where ```public-ip-of-juju-controller``` can be found in ```~/.local/share/juju/controllers.yaml``` in the ```api-endpoints``` section. 
 
-### Restricting log verbosity
-
-Log verbosity in Juju is set at the model level. You can adjust it at any time:  
-
-```
-juju add-model cdk-development --config logging-config='<root>=DEBUG;unit=DEBUG'
-```
-
-and later
-
-```
-juju config-model cdk-production --config logging-config='<root>=ERROR;unit=ERROR'
-```
-
-In addition, the jujud daemon is started in debug mode by default on all controllers. To remove that behavior edit ```/var/lib/juju/init/jujud-machine-0/exec-start.sh``` on the controller node and comment the ```--debug``` section. 
-
-It will then look like: 
-
-```
-#!/usr/bin/env bash
-
-# Set up logging.
-touch '/var/log/juju/machine-0.log'
-chown syslog:syslog '/var/log/juju/machine-0.log'
-chmod 0600 '/var/log/juju/machine-0.log'
-exec >> '/var/log/juju/machine-0.log'
-exec 2>&1
-
-# Run the script.
-'/var/lib/juju/tools/machine-0/jujud' machine --data-dir '/var/lib/juju' --machine-id 0 # --debug
-```
-
-Then restart the service with: 
-
-```
-sudo systemctl restart jujud-machine-0.service
-```
-
-See the [official documentation](https://jujucharms.com/docs/stable/models-config) for more information about logging and other model settings in Juju. 
-
-
 ## Managing your Kubernetes cluster
+
 ### Running privileged containers
 
-By default CDK does not support running privileged containers. If you need them, you have to edit ```/etc/default/kube-apiserver``` on the master nodes, and ```/etc/default/kubelet``` on your worker nodes. 
+By default juju-deployed clusters do not support running privileged containers. If you need them, you have to edit ```/etc/default/kube-apiserver``` on the master nodes, and ```/etc/default/kubelet``` on your worker nodes. 
```

Comment:
Good catch on this one. Is this worth filing a bug about having us support privileged containers as a boolean flag on the worker charm? I can see deploying a separate pool of workers targeted for privileged workloads.

something like:

```
juju deploy cs:~containers/kubernetes-worker dmz-worker
juju config dmz-worker privlieged-containers=true
```

And those DMZ workers can run those escalated privleges away from other workloads, so they can only poke at each other and potentially tank the worker nodes in this known "unrestricted" set of workers.